docs/installation.md

```diff
@@ -28,6 +28,32 @@ Generate all the mocks for your project:
 
 	docker run -v "$PWD":/src -w /src vektra/mockery --all
 
+### pre-commit
+
+Add the following lines to your `.pre-commit-config.yaml` file
+
+```yaml
+repos:
+  - repo: https://github.com/ccoVeille/mockery
```

Comment:
```suggestion
  - repo: https://github.com/vektra/mockery
```

Reply:
I kept it in sync to help mockery mainteners to be able to test

But yes, I will have to change this one